contents/projectManagement/AgileDevelopment.md

```diff
@@ -0,0 +1,127 @@
+# Agile Development
+
+Author: [Joanne Ong](https://github.com/joanneong)
+
+## Overview
+
+In 2015, Forbes published an article boldly titled [Agile: The World's Most Popular Innovation Engine](https://www.forbes.com/sites/stevedenning/2015/07/23/the-worlds-most-popular-innovation-engine/#22b176267c76). Indeed, as a software engineering student, you may have heard of the term "agile development", "agile model" or even simply "agile" before. In this chapter, we will explore agile development in greater depth, and learn why agile development is still thriving today.
+
+## What is agile development?
+
+Popularised in 2001 by the [*Manifesto for Agile Software Development*](http://agilemanifesto.org/), agile development is essentially a software development method that is flexible, iterative and people-centric.
+
+But what does "flexible, iterative and people-centric" really mean? To comprehend this, let us examine each characteristic individually.
+
+---
+
+For a start, agile development is "flexible" because it gives a project more room to evolve over time. This is possible as agile development utilises a form of planning known as [adaptive planning](https://www.projectmanager.com/blog/how-to-plan-in-an-agile-environment), which means that planning is done continuously throughout a project rather than sticking rigidly to a fixed plan established at the start. As a result, planning can take into consideration the project's current progress, and change accordingly to allow the project's priorities and scope to evolve.
+
+To visualise this, let us consider an everyday scenario below:
+
+>Suppose a planner intends to transfer between airplanes, and the planner’s normal plan is to:
```

Comment:
This example seems to illustrate why we should not take a "general plan" and apply it to a novel set of circumstances but does not show how planning is done continuously throughout a project. Maybe it will be better to give an example of a project where the requirements had to change?